Il2CppInterop.Runtime/InteropTypes/Il2CppObjectBase.cs

```diff
@@ -82,10 +82,10 @@ public T Unbox<T>() where T : unmanaged
     }
 
     private static readonly Type[] _intPtrTypeArray = { typeof(IntPtr) };
-    private static readonly MethodInfo _getUninitializedObject = typeof(RuntimeHelpers).GetMethod(nameof(RuntimeHelpers.GetUninitializedObject))!;
+    private static readonly MethodInfo _getUninitializedObject = typeof(FormatterServices).GetMethod(nameof(FormatterServices.GetUninitializedObject))!;
```

Comment:
According to source.dot.net, `FormatterServices.GetUninitializedObject` just calls `RuntimeHelpers.GetUninitializedObject`. Please revert this change and correct the comment instead.